Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM registry.fedoraproject.org/fedora
+
+# For root user
+RUN set -x && \
+  mkdir -p /run/
+
+COPY plans/main.fmf /run/plans/
+COPY .fmf/ /run/.fmf
```

Comment:
What about copying `tests` and the `/plans/provision/container` plan as well? So that users can easily experiment inside the container and the default run (without any volume mount) would do a smoke test?